src/test/java/api/items/ItemApiMoveExamples.java

```diff
@@ -0,0 +1,148 @@
+package api.items;
+
+import static api.support.InstanceSamples.smallAngryPlanet;
+import static org.folio.inventory.support.http.ContentType.APPLICATION_JSON;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+import java.net.MalformedURLException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import org.folio.inventory.domain.items.ItemStatusName;
+import org.folio.inventory.support.http.client.IndividualResource;
+import org.folio.inventory.support.http.client.Response;
+import org.folio.inventory.support.http.client.ResponseHandler;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import api.support.ApiRoot;
+import api.support.ApiTests;
+import api.support.InstanceApiClient;
+import api.support.builders.HoldingRequestBuilder;
+import api.support.builders.ItemRequestBuilder;
+import api.support.builders.ItemsMoveRequestBuilder;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import junitparams.JUnitParamsRunner;
+
+@RunWith(JUnitParamsRunner.class)
+public class ItemApiMoveExamples extends ApiTests {
+
+  private static final String HOLDINGS_RECORD_ID = "holdingsRecordId";
+
+  @Test
+  public void canCreateItemsMove() throws InterruptedException, MalformedURLException, TimeoutException, ExecutionException {
```

Comment:
As I understand it, this isn't creating any new resources or records, it is moving existing items to a different holdings record?

```suggestion
  public void canMoveIItemsToDifferentHoldingsRecord() throws InterruptedException, MalformedURLException, TimeoutException, ExecutionException {
```